### PR TITLE
Release/0.21.1

### PR DESCRIPTION
**Pull request by jmoody**
### 0.21.1

* Coordinates: full-screen pans start and end closer to screen edge #1329
* Support the new view hierarchy of the navigation bar on iOS 11 #1325 @MortenGregersen
* Remove usage tracking to comply with EU GDPR 2018 #1320

### CODE REVIEW

File: calabash-cucumber/lib/calabash-cucumber/ibase.rb [hunk 1]
```diff
@@ -73,7 +73,11 @@ def initialize(world, transition_duration=0.5)
   #  the subclass does not override the `trait` method
   def trait
     raise "You should define a trait method or a title method" unless respond_to?(:title)
-    "navigationItemView marked:'#{self.title}'"
+    if ios_gte_11?
+      "view:'_UINavigationItemView' label marked:'#{self.title}'"
```
Comment (nfrydenholm) [inline calabash-cucumber/lib/calabash-cucumber/ibase.rb]: @MortenGregersen @jmoody are you sure this is the correct query to verify the title when the app is build with SDK 11?
When I build an app with SDK11, the "view:'_UINavigationItemView'" returns []

This is the query I'm using to verify the title
"UINavigationBar label marked:'#{self.title}'"

Comment (MortenGregersen) [reply calabash-cucumber/lib/calabash-cucumber/ibase.rb]: @nfrydenholm It is the query when testing an app built with iOS 10 SDK on iOS 11 Simulator. I haven't tried building an app with iOS 11 SDK and testing against it.

Comment (nfrydenholm) [reply calabash-cucumber/lib/calabash-cucumber/ibase.rb]: Okay. Well, in that case this default trait will start failing again when an app is build with SDK11. But I guess the best practice is not to rely on the default implementation anyway :)

Comment (MortenGregersen) [reply calabash-cucumber/lib/calabash-cucumber/ibase.rb]: @jmoody Do you have any ideas how to check if an app is built with iOS 11 SDK or earlier?

Comment (jmoody) [reply calabash-cucumber/lib/calabash-cucumber/ibase.rb]: You can ask the app for the SDK that was used to built it.

* https://github.com/calabash/calabash-ios-server/blob/develop/calabash/Classes/Utils/LPInfoPlist.h#L6

At first glance, it looks there is no route to return that information.

Comment (MortenGregersen) [reply calabash-cucumber/lib/calabash-cucumber/ibase.rb]: I just tested both an app built with iOS 10 SDK and one built with iOS 11 SDK on an iOS 11 simulator.
It looks like the query `"UINavigationBar marked:'#{self.title}'"` works for both of them.
@jmoody We should use this query instead. Should I make a new pull request, or should it be done in this pull request?

Comment (jmoody) [reply calabash-cucumber/lib/calabash-cucumber/ibase.rb]: Needs to be in a new pull request. :/

Will be available in the next release.

Comment (MortenGregersen) [reply calabash-cucumber/lib/calabash-cucumber/ibase.rb]: I've created a new pull request: https://github.com/calabash/calabash-ios/pull/1333

**Pull request merged**